src/home/fetch-github/fetch-and-display-previews.ts

```diff
@@ -66,6 +66,10 @@ export async function fetchAvatars() {
   const urlPattern = /https:\/\/github\.com\/(?<org>[^/]+)\/(?<repo>[^/]+)\/issues\/(?<issue_number>\d+)/;
 
   const avatarPromises = cachedTasks.map(async (task) => {
+    if (!task.preview.body) {
+      return Promise.resolve();
+    }
```

Comment:
What exactly are the consequences of this? It's a bad idea to make out of scope changes in the pulls.

Reply:
`'task.preview.body' is possibly 'null' or 'undefined'.`

I thought that rather than optional chaining to avoid the error/warning then allowing for the rest of the logic to not pass then resolve the promise that it made more sense to resolve if the body was `null` | `undefined` beforehand.

`fetchAvatars()` is only ever returned after having called `displayGitHubIssues()` first which has 

```ts
if (!preview.body) {
  throw new Error(`Preview body is undefined for task with id: ${preview.id}`);
}
```

body being optional comes from the type not the logic as `displayGitHubIssues()` calls `renderGitHubIssues()` which has `if (!existingIssueIds.has(task.preview.id.toString()))` which then calls `everyNewIssue()` which has `if (!taskPreview.preview.body) {return}` and sets `data-preview-id` meaning every issue will have a body and an ID (relating to your other comment) coming from cachedTasks (I thought). 

The cypress run seems to repeatedly be failing for that specific test case however mine are passing, I'm unsure exactly what's causing the other test to fail.